examples/ota-provider-app/linux/README.md

```diff
@@ -13,19 +13,19 @@ scripts/examples/gn_build_example.sh examples/ota-provider-app/linux out/debug c
+| -q/--queryImageStatus <updateAvailable \| busy \| updateNotAvailable> | Value for the Status field in the QueryImageResponse for the next QueryImageResponse. <br> Status will revert back to updateAvailable on subsequent responses.                                                                                                                                                                                                                                                                                                                                                              |
+| -x/--ignoreQueryImage <num_times_to_ignore>                           | The number of times to ignore the QueryImage Command and not send a response                                                                                                                                                                                                                                                                                                                                                                                                                                                |  |
+| -y/--ignoreApplyUpdate <num_times_to_ignore>                          | The number of times to ignore the ApplyUpdate Request and not send a response                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| -a/--applyUpdateAction <proceed \| awaitNextAction \| discontinue>    | Value for the Action field in the ApplyUpdateResponse for the next ApplyUpdateResponse. <br> Value will revert back to proceed on subsequent responses.                                                                                                                                                                                                                                                                                                                                                                     |
+| -t/--delayedActionTimeSec <time>                                      | Value in seconds for the DelayedActionTime field in the QueryImageResponse and ApplyUpdateResponse. <br> Value will revert back to 0 seconds on subsequent QueryImage Responses and ApplyUpdate Responses.                                                                                                                                                                                                                                                                                                                    |
```

Comment:
Should we make this 0 for subsequent? Wondering what we couldn't test if this was always set as the same value?

Reply:
I asked tester before making this change.  He preferred to use the delayed action time just once to test that functionality, and then revert back to 0 to save time for subsequent tests.